packages/web/docs/src/content/gateway/other-features/security/aws-sigv4.mdx

```diff
@@ -77,6 +77,28 @@ export const gatewayConfig = defineConfig({
 })
 ```
 
+#### Other credential providers
+
+You can use AWS SDK's credential providers from
+[`@aws-sdk/credential-providers`](https://www.npmjs.com/package/@aws-sdk/credential-providers) such
+as SSO support etc.
+
+```ts filename="gateway.config.ts"
+import { fromNodeProviderChain } from '@aws-sdk/credential-providers'
+import { defineConfig } from '@graphql-hive/gateway'
+
+export const gatewayConfig = defineConfig({
+  awsSigV4: {
```

Comment:
note: the field is `awsSigv4` (lowercase `v`) according to the types.

Reply:
Good catch! Fixed!